pysass.cpp

```diff
@@ -303,6 +303,37 @@ static union Sass_Value* _exception_to_sass_error() {
     return retv;
 }
 
+static PyObject* _exception_to_bytes() {
+    /* Grabs a Bytes instance for you to PySass_Bytes_AS_STRING.
+       Remember to Py_DECREF the object later!
+       TODO: This is a terrible violation of DRY, see above.
```

Comment:
yeah, let's resolve this before shipping, should be pretty easy to combine with the above function